presto-main/src/main/java/com/facebook/presto/twitter/logging/QueryScriber.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.twitter.logging;
+
+import com.facebook.presto.event.query.QueryCompletionEvent;
+import com.facebook.presto.event.query.QueryEventHandler;
+import com.google.common.base.Optional;
+import com.twitter.logging.BareFormatter$;
+import com.twitter.logging.Level;
+import com.twitter.logging.QueueingHandler;
+import com.twitter.logging.ScribeHandler;
+import com.twitter.presto.thriftjava.QueryState;
+import io.airlift.log.Logger;
+import org.apache.thrift.TBase;
+import org.apache.thrift.TException;
+import org.apache.thrift.TSerializer;
+
+import java.util.Base64;
+import java.util.logging.LogRecord;
+
+/**
+ * Class that scribes query completion events
+ */
+public class QueryScriber implements QueryEventHandler<QueryCompletionEvent>
+{
+    private static final String SCRIBE_CATEGORY = "test_presto_query_complete";
+    private static final int MAX_QUEUE_SIZE = 1000;
```

Comment:
Once we have a production category I don't see is every needing to change that. Same for queue size, since that's intended for services with real event rates, which we won't have. I don't see our rate going above 10s per minute, which is a trickle. If we find ourselves changing these, let's make them configurable then.